etc/adminhtml/system.xml

```diff
@@ -68,15 +68,15 @@
                     <validate>required-entry</validate>
                     <config_path>payment/razorpay/auto_invoice</config_path>
                 </field>
-                <field id="enable_webhook" translate="label" type="select" sortOrder="5" showInDefault="1" showInWebsite="0" showInStore="0">
+                <field id="enable_webhook" translate="label" type="select" sortOrder="5" showInDefault="0" showInWebsite="0" showInStore="0">
                     <label>Webhook Enabled</label>
                     <comment>** {{base_url}} Copy the Webhook Url from below **</comment>
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                     <frontend_model>Razorpay\Magento\Model\WebhookUrl</frontend_model>
                     <validate>required-entry</validate>
                     <config_path>payment/razorpay/enable_webhook</config_path>
                 </field>
-                <field id="webhook_secret" translate="label" type="text" sortOrder="5" showInDefault="1" showInWebsite="0" showInStore="0">
+                <field id="webhook_secret" translate="label" type="text" sortOrder="5" showInDefault="0" showInWebsite="0" showInStore="0">
```

Comment:
As we enabling webhook by default , why we needs these configs here ?

Reply:
@ramth05 config settings fields are disabled within form by updating flag `showInDefault` to `0`. Should I remove this code from here?